api/src/main/java/org/eclipse/microprofile/lra/client/LRAClient.java

```diff
@@ -44,40 +44,16 @@
     String LRA_HTTP_RECOVERY_HEADER = "Long-Running-Action-Recovery";
 
     /**
-     * Key for looking up the config property that specifies which host a
-     * coordinator is running on
+     * The config property that specifies a URL exposed for LRA orchestration and
+     * management. An implementation is not required to make use of this property.
      */
```

Comment:
This bit of config is so that the TCK knows when the implementation has recovered. We briefly discussed this yesterday ("Are we going to specify recovery semantics in the spec? ")

We said that we would propose and SPI that implementations would implement in order to pass the TCK. In the LRA hangout a week last Monday I proposed something like:

```java
public ManagementSPI {
LRAInfo getStatus(LRA lra); // get status of one LRA
Collection<LRAInfo> getLRAs(CompensatorStatus status); // list LRAs in the given state
//anything else
}
```
So to get recovering LRAs the status would be set to Completing or Compensating.

This would mean we would not need to configure the endpoint in the TCK.